db/schema.rb

```diff
@@ -322,6 +322,17 @@
     t.index ["slug"], name: "index_classified_listing_categories_on_slug", unique: true
   end
 
+  create_table "classified_listing_categories", force: :cascade do |t|
```

Comment:
isn't this already present on master?

Reply:
Yeah, my local branch must have gotten messed up during a rebase or something. Thanks!